### PR TITLE
libtitan_sys: upgrade libz-sys's version to 1.1.2

### PR DESCRIPTION
**Pull request by gentcys**
Signed-off-by: gentcys <iamchencong@gmail.com>

An error occurred while compiling on macOS, which described in https://github.com/tikv/tikv/issues/8659.
And it's fixed by https://github.com/rust-lang/libz-sys/commit/81026e996eb3804acf1e6898a3c1d791cdfb59f1.
This PR just update the version of dependency `libz-sys`.

### CODE REVIEW

File: librocksdb_sys/libtitan_sys/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ links = "titan"
 [dependencies]
 bzip2-sys = "0.1.8+1.0.8"
 libc = "0.2.11"
-libz-sys = { version = "1.0.25", features = ["static"] }
+libz-sys = { version = "1.1.2", features = ["static"] }
```
Comment (yiwu-arbug) [inline librocksdb_sys/libtitan_sys/Cargo.toml:10]: shall we just specify "1.1"?

**Comment by yiwu-arbug** [comment]
you also need to update librocksdb_sys/Cargo.toml

File: librocksdb_sys/libtitan_sys/Cargo.toml (hunk 1 above)
Comment (gentcys) [reply librocksdb_sys/libtitan_sys/Cargo.toml:10]: Sure! `1.1` is ok because `1.1.2` was released.

**Comment by gentcys** [comment]
> you also need to update librocksdb_sys/Cargo.toml

Ah! i missed it.

**Review by Connor1996 — APPROVED** [review]
Thansk! LGTM

**Pull request merged**